x/gov/test_common.go

```diff
@@ -0,0 +1,105 @@
+package gov
+
+import (
+	"bytes"
+	"log"
+	"sort"
+	"testing"
+
+	abci "github.com/tendermint/abci/types"
+	crypto "github.com/tendermint/go-crypto"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/mock"
+	"github.com/cosmos/cosmos-sdk/x/bank"
+	"github.com/cosmos/cosmos-sdk/x/stake"
```

Comment:
can we get away from importing x/stake here and only use types.Validator?

Reply:
I switched functionality stuff to use types.ValidatorSet, but for testing I still need to use the stakeKeeper.